src/analyses/ai.h

```diff
@@ -543,24 +513,107 @@ class concurrency_aware_ait:public ait<domainT>
   {
   }
 
-  bool merge_shared(
+  virtual bool merge_shared(
```

Comment:
Not really.  I mean, ideally, `concurrency_aware_ait` would be deprecated and replaced with a solution that would work with other domains.  With the new storage it should be pretty easy to merge the relevant locations and VSD's three-way-merge would allow concurrent AI to be done properly.  But, right now, I don't have the time to refactor all of that.

I made it virtual so that it is consistent with the merge method in ai_baset.